query/src/main/kotlin/com/linecorp/kotlinjdsl/query/spec/expression/NestedColumnSpec.kt

```diff
@@ -0,0 +1,35 @@
+package com.linecorp.kotlinjdsl.query.spec.expression
+
+import com.linecorp.kotlinjdsl.query.spec.Froms
+import javax.persistence.criteria.*
+
+private fun <T> ColumnSpec<T>.outerPath(froms: Froms): Path<T> = froms[entity].get(path)
```

Comment:
I think it would be better not to declare this at the file level. This is because it looks like a method to be used only within the NestedColumnSpec class rather than a method to be used in common within this file.

Therefore, I think it would be better to put it inside NestedColumnSpec.

And because you used the word `Nested` to express this business, it would be better to rename the method to nestedPath.

Reply:
I changed, that was good feedback